documentation/meta/release_and_deployment/runbooks/catalog.md

```diff
@@ -6,8 +6,10 @@ For more information on how deployments work, please see the [general deployment
 
 ## Setup
 
-Check [the running DAGs](https://airflow.openverse.org/home?status=running) in
-Airflow to make sure no DAGs are running.
+Check [the running DAGs][dags_running] in Airflow to make sure no DAGs are
+running.
+
+[dags_running]: https://airflow.openverse.org/home?status=active&lastrun=running
```

Comment:
@dhruvkb The rendered document modifies the ampersand in the link, preventing it from going to the correct place. Do you know how to work around this?

Reply:
I don't know for sure if these workarounds will work, and I can try them out if I have some time soon.

1. Writing the `&` encoded as `%26`.
2. Trying a regular HTML URL `<a href="...">the running DAGs</a>` instead of Markdown.

Reply:
Verified that this is an upstream issue in MyST https://github.com/executablebooks/MyST-Parser/issues/760 and the workaround while it is fixed will be to use option 2 i.e. raw HTML. Option 1 doesn't work as I had hoped.

Reply:
```suggestion
Check
<a href="https://airflow.openverse.org/home?status=active&lastrun=running">the
running DAGs</a> in Airflow to make sure no DAGs are running.
```